src/libaktualizr/package_manager/packagemanagerinterface.cc

```diff
@@ -67,6 +72,15 @@ static int ProgressHandler(void* clientp, curl_off_t dltotal, curl_off_t dlnow,
   if (ds->progress_cb && progress > ds->last_progress) {
     ds->last_progress = progress;
     ds->progress_cb(ds->target, "Downloading", progress);
+    //OTA-4864:Improve binary file download progress logging. Report each XX sec report event that notify user
+    if (!ds->target.IsOstree()) {      
+      auto now = std::chrono::steady_clock::now();
+      auto milliseconds = std::chrono::duration_cast<std::chrono::milliseconds>(now - ds->time_lastreport);
+      if(milliseconds.count() > LogProgressInterval || progress == 100) {
+        LOG_INFO << "Download progress for file " << ds->target.filename() <<": " << progress << std::endl;
```

Comment:
No need for `std::endl`; the logger already handles newlines automatically.

Reply:
removed